pkg/scheduler/objects/application.go

```diff
@@ -695,44 +697,60 @@ func (sa *Application) addAllocationAskInternal(ask *AllocationAsk) {
 	}
 }
 
-func (sa *Application) UpdateAskRepeat(allocKey string, delta int32) (*resources.Resource, error) {
+func (sa *Application) AllocateAsk(allocKey string) (*resources.Resource, error) {
```

Comment:
It seems this method is used by testing only, so should we keep it in production code?

Reply:
It's needed (unfortunately) as there are multiple test packages that call it (pkg/scheduler and pkg/scheduler/objects). Therefore it needs to be here in order to be available to both.